defaults/main.yml

```diff
@@ -29,7 +29,6 @@ firewalld_default_interface_zone: public
 iptables_service: iptables
 manage_firewall_rules: true
 openvpn_firewall: auto
-openvpn_masquerade_not_snat: false
```

Comment:
looks like we need this variable, it still using here:
https://github.com/aovpn/ansible-role-openvpn/blob/main/tasks/ufw.yml#L44

Reply:
Just removed all inclusions of this variable: from iptables, from ufw and from readme file.